core/src/main/java/org/sonarsource/sonarlint/core/analyzer/issue/DefaultClientIssue.java

```diff
@@ -33,16 +33,18 @@
 
 public final class DefaultClientIssue extends TextRangeLocation implements org.sonarsource.sonarlint.core.client.api.common.analysis.Issue {
   private final String severity;
+  private final String type;
   private final ActiveRule activeRule;
   private final String primaryMessage;
   private final ClientInputFile clientInputFile;
   private final Rule rule;
   private final List<Flow> flows;
 
-  public DefaultClientIssue(String severity, ActiveRule activeRule, Rule rule, String primaryMessage, @Nullable TextRange textRange,
+  public DefaultClientIssue(String severity, @Nullable String type, ActiveRule activeRule, Rule rule, String primaryMessage, @Nullable TextRange textRange,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)